src/commands.rs

```diff
@@ -363,19 +363,27 @@ fn print_storage_status(
 }
 
 /// Query and pretty print the status that is common to all Nitrokey devices.
-fn print_status(ctx: &mut Context<'_>, device: &nitrokey::DeviceWrapper<'_>) -> anyhow::Result<()> {
+fn print_status(
+  ctx: &mut Context<'_>,
+  idx: usize,
+  path: &str,
+  device: &nitrokey::DeviceWrapper<'_>,
+) -> anyhow::Result<()> {
   let serial_number = device
     .get_serial_number()
     .context("Could not query the serial number")?;
 
   println!(
     ctx,
-    r#"Status:
+    r#"Device #{idx}:
+  USB path:          {path}
```

Comment:
Would you mind creating a separate change for the printing of the path? Or does that not make sense?

Reply:
Yes, I can do that.